components/Carbon.js

```diff
@@ -377,13 +376,21 @@ function useHighlightLoader() {
   }, [])
 }
 
-function selectedLinesReducer({ prevLine, selected }, { type, lineNumber, numLines }) {
+function selectedLinesReducer(
+  { prevLine, selected },
+  { type, lineNumber, numLines, selectedLines }
+) {
   const newState = {}
 
   if (type === 'GROUP' && prevLine) {
     for (let i = Math.min(prevLine, lineNumber); i < Math.max(prevLine, lineNumber) + 1; i++) {
       newState[i] = selected[prevLine]
     }
+  }
+  if (type === 'MULTILINE') {
```

Comment:
Hmm what is `MULTILINE` used for? Just for selecting  any arbitrary list of lines? 🙂

Reply:
Yes, that was my idea to support selecting a list of lines. Do you have a better suggestion for the name? In theory the other two actions GROUP and LINE could be also expressed as MULTILINE but I didn't want to change to much.